src/workloads/execution/linearFill.yml

```diff
@@ -0,0 +1,308 @@
+        allowDiskUse: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+
+- Name: LinearFillMultipleOutputWithSinglePartition
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Duration: 30 seconds
+    Database: *db
+    Operations:
+    - OperationMetricsName: TestMultipleOutputWithSinglePartition
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [{
+          $setWindowFields: {
+            sortBy: {sortDateField: 1},
+            partitionBy: "$part1",
+            output: {
+              numeric: {
+                $linearFill: "$numeric"},
+              integer: {
+                $linearFill: "$integer"},
+            }
+          }
+        }]
+        cursor: {batchSize: *batchSize}
+        allowDiskUse: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+
+- Name: LinearFillWithMultiplePartitions
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Duration: 30 seconds
+    Database: *db
+    Operations:
+    - OperationMetricsName: TestSingleOutputWithMultiplePartitions
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [{
+          $setWindowFields: {
+            sortBy: {sortField: 1},
+            partitionBy: {part1: "$part1", part2: "$part2"},
+            output: {
+              numeric: {
+                $linearFill: "$numeric"}
+            }
+          }
+        }]
+        cursor: {batchSize: *batchSize}
+        allowDiskUse: true
+  - Nop: true
+  - Nop: true
+
+- Name: LinearFillMultipleOutputWithMultiplePartitions
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Nop: true
+  - Duration: 30 seconds
+    Database: *db
+    Operations:
+    - OperationMetricsName: TestMultipleOutputWithMultiplePartitions
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline: [{
+          $setWindowFields: {
+            sortBy: {sortDateField: 1},
+            partitionBy: {part1: "$part1", part2: "$part2"},
+            output: {
+              numeric: {
+                $linearFill: "$numeric"},
+              double: {
+                $linearFill: "$double"},
+            }
+          }
+        }]
+        cursor: {batchSize: *batchSize}
+        allowDiskUse: true
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - standalone
+      - replica
+    branch_name:
+      $neq:
+      - v4.0
+      - v4.2
+      - v4.4
+      - v5.0
+      - v5.1
```

Comment:
Also don't want to run on 5.2 right?

Reply:
I guess we don't want to run on 5.2 because the purpose of the perf test is to not use a feature flag - right? Bc linearFill did get in yesterday before the release was cut

Reply:
Right, the feature flag would have to have been removed